src/devices/AD5328/AD5328.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Spi;
+using UnitsNet;
+
+namespace Iot.Device.DAC
+{
+    /// <summary>
+    /// Driver for the AD5328 DAC.
+    /// </summary>
+    public class AD5328 : IDisposable
+    {
+        private SpiDevice _spiDevice;
+        private ElectricPotential _referenceVoltageA;
+        private ElectricPotential _referenceVoltageB;
+        private bool _disposedValue;
+
+        /// <summary>
+        /// Initializes a new instance of the AD5328 device.
+        /// </summary>
+        /// <param name="spiDevice">The SPI device used for communication.</param>
+        /// <param name="referenceVoltageA">The reference voltage for the first 4 channels</param>
+        /// <param name="referenceVoltageB">The refrence volatage for the last 4 channels</param>
+        public AD5328(SpiDevice spiDevice, ElectricPotential referenceVoltageA, ElectricPotential referenceVoltageB)
+        {
+            _spiDevice = spiDevice;
+            _referenceVoltageA = referenceVoltageA;
+            _referenceVoltageB = referenceVoltageB;
+        }
+
+        /// <summary>
+        /// Sets the voltage of a certain channel
+        /// </summary>
+        /// <param name="channel">The channel number. Zero based. channel A = 0</param>
+        /// <param name="voltage">The voltage</param>
+        public void SetDACVoltage(UInt16 channel, ElectricPotential voltage)

```

Comment:
I'm wondering if it's important to have DAC in the name? Would SetVoltage be enough?

Reply:
fixed!